.github/workflows/build_wheel.py

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env python3
+
+import os
+import sys
+from pathlib import Path
+import subprocess
+
+ROOT = Path(__file__).parent.parent.parent
+
+# For macOS and Windows and linux, run Maturin against the Python interpreter that's
+# been installed and configured for this CI run, i.e. the one that's running
+# this script.
+# Note the docker image konstin2/maturin:master does not work.
+
+os.chdir("./")
+if len(sys.argv) > 1 :
+    command = sys.argv[1].split(",")
+    command.append(sys.executable)
+else:
+    command = [
+        "maturin",
+        "build",
+        "--release",
+        "--compatibility=manylinux_2_35",
+        "--interpreter",
+        sys.executable,
+    ]
+subprocess.run(command, check=True)
+os.chdir("..")
```

Comment:
This line (line 29) can be removed as it is causing the build issue of FileNotFound Error for python 3.8 and earlier.

Reply:
Done